src/features/presence/PresenceAvatar.js

```diff
@@ -34,20 +51,4 @@ const PresenceAvatar = ({
     </div>
 );
 
-PresenceAvatar.propTypes = {
-    avatarUrl: PropTypes.string,
-    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    isActive: PropTypes.bool,
-    name: PropTypes.string.isRequired,
-    onMouseEnter: PropTypes.func,
-    onMouseLeave: PropTypes.func,
-    onFocus: PropTypes.func,
-    onBlur: PropTypes.func,
-    isDropDownAvatar: PropTypes.bool,
-};
-
-PresenceAvatar.defaultProps = {
-    isActive: false,
-};
-
-export default PresenceAvatar;
+export default injectIntl(PresenceAvatar);
```

Comment:
Do we have any cases where we're using the ref of this component? If so, do they still work properly?

Reply:
Good callout, I poked around a few other repos and didn't find any references so we should be good here